internal/status/status.go

```diff
@@ -0,0 +1,130 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package status
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowcli/project"
+	"github.com/onflow/flow-cli/pkg/flowcli/services"
+	"github.com/spf13/cobra"
+)
+
+type FlagsStatus struct {
+}
+
+var statusFlags = FlagsStatus{}
+
+// Result structure
+type Result struct {
+	network         string
+	accessNode      string
+	connectionError error
+}
+
+const (
+	OnlineIcon   = "🟢"
+	OnlineStatus = "ONLINE"
+
+	OfflineIcon   = "🔴"
+	OfflineStatus = "OFFLINE"
+)
+
+var InitCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:   "status",
+		Short: "Display status of network",
+	},
+	Flags: &statusFlags,
+	Run: func(
+		cmd *cobra.Command,
+		args []string,
+		globalFlags command.GlobalFlags,
+		services *services.Services,
+	) (command.Result, error) {
+		// Get network name from global flag
+		network := globalFlags.Network
+		if network == "" {
+			network = "emulator"
+		}
+
+		proj, err := project.Load(globalFlags.ConfigPath)
```

Comment:
Project is already loaded in services, use the loaded project in service layer and return the values you want with multiple returns.